packages/kbn-plugin-generator/sao_template/template/eslintrc.js

```diff
@@ -1,24 +1,9 @@
-module.exports = {
-  root: true,
+module.exports = {	
```

Comment:
When generator is run with `-i` this file `.eslintrc.js` is added to Kibana internal plugin, not sure if that is expected.

Reply:
fixed